### PR TITLE
chore: updated readme file to run the application locally.

### PR DESCRIPTION
**Pull request by navdeepg2021**
**Related Issue(s):** #

**Proposed Changes:**

1.
2.

**PR Checklist:**

- [x] This PR is made against the dev branch (all proposed changes except releases should be against dev, not main).
- [x] This PR has **no** breaking changes.
- [x] This PR contains only one commit.
- [ ] I have updated or added new tests to cover the changes in this PR.
- [ ] I have updated the version in `package.json` to follow the [Semantic Versioning](https://semver.org/) guidelines.
- [ ] All tests are passing.
- [ ] I have added my changes to the [CHANGELOG](https://github.com/source-cooperative/source.coop/blob/dev/CHANGELOG.rst)
      **or** a CHANGELOG entry is not required.
- [ ] This PR affects the [Source Cooperative Data Proxy](https://github.com/source-cooperative/data.source.coop),
      and I have opened issue/PR #XXX to track the change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This repository contains the Next.JS application which hosts the Source Cooperat
 
 ### Install Dependencies
 
-To install the dependencies, run the following command:
+To install the dependencies, run the following command for main project and the Source Cooperative CLI project:
```
Comment (kbgg) [inline README.md:13]: Change to "To install the dependencies for the Frontend, API, and CLI run the following command:"

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Next, copy the `API endpoint` URL and set it as the `ORY_SDK_URL` environment va
 
 ### Run Locally
 
-After having set up your environment variables, run the following command to start the service locally:
+After having set up your environment variables, run the following command to start the service locally after completing the above steps:
```
Comment (kbgg) [inline README.md:36]: Change to "After completing the steps above and setting up your environment variables, run the following command to start the service locally:"

**Review by kbgg — CHANGES_REQUESTED** [review]

**Comment by navdeepg2021** [comment]
@kbgg There is a feature where you can make suggestions in such a way where the author has to just accept the suggestion and directly commit. I would appreciate that if you are so opinionated about the language for readme file also and respect each other's time.

**Comment by kbgg** [comment]
> @kbgg There is a feature where you can make suggestions in such a way where the author has to just accept the suggestion and directly commit. I would appreciate that if you are so opinionated about the language for readme file also and respect each other's time.

That feature is seemingly not enabled for this repository, feel free to enable it and I'll make those changes for you. Otherwise, feel free to request a review from someone else who is less opinionated about low quality PRs with grammatical errors.

**Comment by navdeepg2021** [comment]
> > @kbgg There is a feature where you can make suggestions in such a way where the author has to just accept the suggestion and directly commit. I would appreciate that if you are so opinionated about the language for readme file also and respect each other's time.
> 
> That feature is seemingly not enabled for this repository, feel free to enable it and I'll make those changes for you. Otherwise, feel free to request a review from someone else who is less opinionated about low quality PRs with grammatical errors.

Thanks for your opinions.

**Pull request closed without merging**